src/routes/PostDetail/Comments/CommentForm/CommentForm.js

```diff
@@ -51,33 +55,40 @@ export default class CommentForm extends Component {
   render () {
     const { currentUser, className, addAttachment } = this.props
 
-    if (!currentUser) return null
-
-    const placeholder = `Hi ${currentUser.firstName()}, what's on your mind?`
+    const placeholder = currentUser ? `Hi ${currentUser.firstName()}, what's on your mind?` : "Hi! What's on your mind?"
 
     return <div
       styleName='commentForm'
       className={className}
       onClick={() => this.editor.current.focus()}>
       <AttachmentManager type='comment' id='new' />
```

Comment:
Probably shouldn't render this either in the case of `!currentUser` though I think it's largely immaterial.

Reply:
actually it looks nice to have the comment box there but unusable and with a button that says Sign up to comment

Reply:
Oh for sure, in total agreement. I mean the AttachmentManager is not necessary to render (there is nothing visible about it when there are no attachments)

Reply:
oh right